units/XNA0303/XNA0303_unit.bp

```diff
@@ -107,6 +107,7 @@ UnitBlueprint {
         'ASF',
     },
     CollisionOffsetY = 0,
+    CollisionOffsetZ = -0.2,
```

Comment:
these lines dont change anything since ASF get a spherical collision mesh upon take off so all collider information is overridden. They should stay as is since they fit the model better.